src/js/components/Box/__tests__/Box-test.js

```diff
@@ -298,6 +318,26 @@ describe('Box', () => {
         <Box pad={{ start: 'small' }} />
         <Box pad={{ end: 'small' }} />
         <Box pad={{ top: 'small' }} />
+        <Box pad={{ top: 'small', left: 'medium', horizontal: 'large' }} />
+        <Box pad={{ top: 'small', vertical: 'large' }} />
+        <Box
+          pad={{
+            top: 'small',
+            right: 'medium',
+            horizontal: 'small',
+            vertical: 'large',
```

Comment:
can we test a use-case for vertical===horizontal? and also when all the sides are equal? I'm trying to match the tests to the approach taken on the padding PR #3939

Reply:
Yes. Thanks, let me add those.

Reply:
Done!